docs/setup-guide/arktos-with-mizar-cni.md

```diff
@@ -0,0 +1,45 @@
+# Arktos deployment with Mizar CNI 
+
+The machines in this lab used are AWS EC2 t2-large (2 CPUs, 8GB mem), Ubuntu 18.04 LTS.
+
+The steps might change with the progress of development.
+
+1. To install dependencies required for Arktos, run the following commands: 
+```bash
+wget https://raw.githubusercontent.com/CentaurusInfra/arktos/master/hack/setup-dev-node.sh
```

Comment:
Why do you guys need to introduce these manual steps? Since you are going to start arktos-up in the clone of the repo, you can run this without wget. I have repeatedly mentioned we need to reduce manual steps. You guys just don't get it, do you?